apps/coscrad-frontend-e2e/src/e2e/resources/terms/term.query-flow.e2e.cy.ts

```diff
@@ -27,13 +27,13 @@ describe(`Term index-to-detail flow`, () => {
         });
     });
 
-    describe(`the resource menu`, () => {
+    describe(`the resource types page`, () => {
         beforeEach(() => {
             cy.visit(`/Resources`);
         });
 
         it('should have an entry for terms', () => {
-            cy.contains('terms');
+            cy.contains('Terms');
```

Comment:
Can we make this case-insensitive instead?

Reply:
The reason I changed this was because it was catching "terms" in the description of the vocabulary list blurb, not the link to the Terms themselves.  So the test was passing even though the Terms link could have been missing.